src/app/my-rewards/builder/components/AllTimeShare.tsx

```diff
@@ -0,0 +1,45 @@
+import { CompleteBuilder } from '@/app/collective-rewards/types'
+import { filterBuildersByState, useBuilderContext } from '@/app/collective-rewards/user'
+import { useHandleErrors } from '@/app/collective-rewards/utils'
+import { Span } from '@/components/TypographyNew/'
+import { TOKENS } from '@/lib/tokens'
+import { Address } from 'viem'
+import { useBuilderAllTimeShare } from '@/app/my-rewards/builder/hooks/useBuilderAllTimeShare'
+import { RewardCard } from '@/app/my-rewards/components/RewardCard'
+
+export interface RewardCardAllTimeShareProps {
+  isLoading: boolean
+  amount: string
+}
+
+export const AllTimeShare = ({ gauge }: { gauge: Address }) => {
+  const { builders, isLoading: isBuildersLoading, error: buildersError } = useBuilderContext()
+  const activatedBuilders = filterBuildersByState<CompleteBuilder>(builders, {
+    activated: true,
+  })
```

Comment:
```suggestion
  const activatedBuilders = filterBuildersByState<CompleteBuilder>(builders)
```
We need all the ones that have gauges.

Reply:
let's have a discussion about this